File: Uri-Online-Judge/Wellington/C:C++/Iniciante/1043.cpp
#include <stdio.h>
#include <stdlib.h>
#include <cmath>

int main(){
    double a, b, c;
    scanf("%lf", &a);
    scanf("%lf", &b);
    scanf("%lf", &c);
    
    if (std::abs(b-c) < a && a < b+c &&
        std::abs(a-c) < b && b < a+c &&
        std::abs(a-b) < c && c < a+b) {
        printf("Perimetro = %.1lf\n", a+b+c);
    } else {
        printf("Area = %.1lf\n", (c * (a+b))/2);
    }
    
    return 0;
}

File: Uri-Online-Judge/Wellington/Python 3/Iniciante/1043.py
# -*- coding: utf-8 -*-

linha = list(map(float, input().split()))
a, b, c = linha

if abs(b-c) < a and a < b+c and abs(a-c) < b and b < a+c and abs(a-b) < c and c < a+b:
	print('Perimetro = {:.1f}'.format(a+b+c))
else:
	print('Area = {:.1f}'.format((c * (a + b))/2))